ext/mysqli/tests/mysqli_stmt_execute_bind.phpt

```diff
@@ -0,0 +1,154 @@
+--TEST--
+mysqli_stmt_execute() - bind in execute
+--SKIPIF--
+<?php
+require_once('skipif.inc');
+require_once('skipifconnectfailure.inc');
+if (!$link = my_mysqli_connect($host, $user, $passwd, $db, $port, $socket)) {
+    die(sprintf('skip Cannot connect to MySQL, [%d] %s.', mysqli_connect_errno(), mysqli_connect_error()));
+}
+if (mysqli_get_server_version($link) <= 40100) {
```

Comment:
I think we can safely drop checks for MySQL 4.1, nobody is testing PHP 8.1 with this MySQL version.